guide/voice/the-basics.md

```diff
@@ -0,0 +1,123 @@
+# The Basics
+
+Now that you have your dependencies installed, you can start using voice!
+
+## Joining voice channels
+
+Joining a voice channel is easy, here's a common example:
+
+```js
+client.on('message', async message => {
+	// Join the same voice channel of the author of the message
+	if (message.member.voice.channel) {
+		const connection = await message.member.voice.channel.join();
+	}
+});
+```
+
+If you try to join a channel the bot is already in, nothing will happen so we don't need to worry about running this method repeatedly.
+
+::: tip
+In the above example we accessed a `VoiceState` through `member.voice`. A voice state exists for members connected to a voice channel, and it tells you which channel they're connected to, if the member is speaking, and whether they're deafened or muted. Consult the documentation for a full description of what voice states can do!
```

Comment:
Consider leaving out the "it" of "and it tells you" here, although it's purely a choice of preference. To me leaving out the "it" sounds a bit more fluid but ymmv.